emmet-core/emmet/core/molecules/summary.py

```diff
@@ -0,0 +1,379 @@
+                    "(Mulliken, Restrained Electrostatic Potential, Natural Bonding Orbitals, etc.)"
+    )
+
+    # partial spins
+    partial_spins: Dict[str, List[float]] = Field(
+        None,
+        description="Atomic partial spins for the molecule using different partitioning schemes "
+                    "(Mulliken, Natural Bonding Orbitals, etc.)"
+    )
+
+    # bonding
+    molecule_graphs: Dict[str, MoleculeGraph] = Field(
+        None,
+        description="Molecular graph representations of the molecule using different "
+                    "definitions of bonding."
+    )
+
+    bond_types: Dict[str, Dict[str, List[float]]] = Field(
+        description="Dictionaries of bond types to their length under different "
+                    "definitions of bonding, e.g. C-O to a list of the lengths of "
+                    "C-O bonds in Angstrom."
+    )
+
+    bonds: Dict[str, List[Tuple[int, int]]] = Field(
+        description="List of bonds under different definitions of bonding. Each bond takes "
+                    "the form (a, b), where a and b are 0-indexed atom indices",
+    )
+
+    bonds_nometal: Dict[str, List[Tuple[int, int]]] = Field(
+        description="List of bonds under different definitions of bonding with all metal ions "
+                    "removed. Each bond takes the form in the form (a, b), where a and b are "
+                    "0-indexed atom indices.",
+    )
+
+    # redox properties
+    electron_affinity: float = Field(None, description="Vertical electron affinity in eV")
+
+    ea_id: MPID = Field(None, description="MPID for electron affinity")
+
+    ionization_energy: float = Field(None, description="Vertical ionization energy in eV")
+
+    ie_id: MPID = Field(None, description="MPID for ionization energy")
+
+    reduction_free_energy: float = Field(None, description="Adiabatic free energy of reduction")
+
+    red_id: MPID = Field(None, description="MPID for adiabatic reduction")
+
+    oxidation_free_energy: float = Field(None, description="Adiabatic free energy of oxidation")
+
+    ox_id: MPID = Field(None, description="MPID for adiabatic oxidation")
+
+    reduction_potentials: Dict[str, float] = Field(None,
+                                                   description="Reduction potentials with various "
+                                                               "reference electrodes")
+
+    oxidation_potentials: Dict[str, float] = Field(None,
+                                                   description="Oxidation potentials with various "
+                                                               "reference electrodes")
+
+    # has props
+    has_props: List[HasProps] = Field(
+        None, description="List of properties that are available for a given material."
+    )
+
+    @classmethod
+    def from_docs(cls, molecule_id: MPID, **docs: Dict[str, Union[Dict, List[Dict]]]):
+        """Converts a bunch of summary docs into a SummaryDoc"""
```

Comment:
This sounds like a tautology - are the input docs, e.g. `MoleculeDoc`, etc. or are they actually other instances of `SummaryDoc`? It isn't clear to me how you would turn multiple `SummaryDoc` into one `SummaryDoc`.

Maybe just say "Converts a set of property documents into a SummaryDoc"